### ISSUE TITLE
Skip to main content link doesn't always work

### ISSUE THREAD

**Issue by starsplatter** [samvera-labs/nurax-pre2023#226]
### Descriptive summary
Sometimes clicking the skip to main content link doesn't work. It generally won't work for me the first time i try it. I thought maybe it was related to url formation but it's really inconsistent when it happens. I wonder if turbo links is being bad?

### Rationale

Keyboard users need to the skip to main content link

### Expected behavior
Clicking skip to main content skips to the main content

### Actual behavior
The page does not move

### Steps to reproduce the behavior

1. Go to any page and try using the skip to main content link

### Related work

An issue for skip to main content link not displaying focus already exists in the hyrax repo.

**Comment by vantuyls** [comment]
related issues:
https://github.com/samvera/hyrax/issues/498
https://github.com/samvera/hyrax/issues/497
